packages/pg/lib/utils.js

```diff
@@ -52,7 +54,7 @@ var prepareValue = function (val, seen) {
     }
     return buf.slice(val.byteOffset, val.byteOffset + val.byteLength) // Node.js v4 does not support those Buffer.from params
   }
-  if (val instanceof Date) {
+  if (val instanceof Date || util.types.isDate(val)) {
```

Comment:
I think there should be a better justification for odd code than just “in case”. Every change can break something contrived; here, it’s things like `Object.create(Date.prototype)` and `Object.defineProperty(Date, Symbol.hasInstance, …)` that don’t seem worth worrying about.

```suggestion
  if (util.types.isDate(val)) {
```

Reply:
Thanks for the feedback, I've reverted that back to what I had originally for the change to just use `isDate`